baseplate/experiments/providers/variant_sets/multi_variant_set.py

```diff
@@ -0,0 +1,65 @@
+from .base import VariantSet
+
+
+class MultiVariantSet(VariantSet):
+    """ Variant Set designed to handle more than two total treatments.
+
+    MultiVariantSets are not designed to support changes in variant sizes without
+    rebucketing.
+    """
+
+    def __init__(self, variants, num_buckets=1000):
+        """ :param variants -- array of dicts, each containing the keys 'name'
+            and 'size'. Name is the variant name, and size is the fraction of
+            users to bucket into the corresponding variant. Sizes are expressed
+            as a floating point value between 0 and 1.
+        :param num_buckets -- the number of potential buckets that can be
```

Comment:
Can you add the `type`s to these `:param` declarations?  Also, same formatting comments as the docstrings in `simple_experiment.py`

Reply:
fixed.